### ISSUE TITLE
esp32-smartdisplay

### ISSUE THREAD

**Issue by GrahamM** [witnessmenow/ESP32-Cheap-Yellow-Display#34]
There's an existing library at https://github.com/rzeldent/esp32-smartdisplay that makes using the 2432S024R/C/N, 2432S028R, 3248S035R/C and 8048S070N/C so much easier for getting things up and running quickly. It supports LVGL and enabled me to port an existing app targeting an M5 stack to a 3248S035C in less than 20 minutes.

It might be worth considering working with the author of that project, given the boards it supports.

**Comment by witnessmenow** [comment]
Yes I must try it out, seems like the cyd is already something it supports!
